motor_control/throttle/button_control.ino

```diff
@@ -1,29 +1,23 @@
-#include "button_control.h"
-
-ButtonThrottle::ButtonThrottle(const int &pin, int timeToFull) : Throttle(pin, true)
-{
-  this->m_timeToFull = timeToFull;
-  m_isPressed = false;
-}
-
 int ButtonThrottle::mappingFunction(int time_pressed, m_map_type mapType)
 {
-  if (mapType == LINEAR)
+  if(m_isPressed == true)
+  {
+if (mapType == LINEAR)
```

Comment:
Might want to fix the indentation here, it's kind of confusing to read otherwise

Reply:
agreed